app/components/site_search_form_component.html.erb

```diff
@@ -1,6 +1,6 @@
-<%= form_tag @url, method: :get, class: 'search-across-form search-query-form row gx-0', role: 'search', 'aria-label' => t('blacklight.search.form.submit') do %>
-  <div class="col-auto search-q"></div>
-  <div class="col-auto">
+<%= form_tag @url, method: :get, class: 'search-across-form search-query-form row gx-0 flex-nowrap', role: 'search', 'aria-label' => t('blacklight.search.form.submit') do %>
```

Comment:
Is `flex-nowrap` required here?  Isn't that the default?

Reply:
It isn't the default, but this is cruft from another solution. Good catch.